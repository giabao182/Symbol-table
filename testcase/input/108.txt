INSERT a number
INSERT b string
BEGIN
INSERT a string
INSERT c number
PRINT
END
PRINT