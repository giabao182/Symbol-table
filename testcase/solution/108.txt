success
success
Redeclared: INSERT a string
success
a//0 b//0 c//1
a//0 b//0