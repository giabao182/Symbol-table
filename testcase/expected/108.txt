success
success
success
success
b//0 a//1 c//1
a//0 b//0